docs/crio.8.md

```diff
@@ -289,7 +290,9 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--version, -v**: print the version
 
-**--version-file**="": Location for CRI-O to lay down the version file (default: /var/run/crio/version)
+**--version-file**="": Location for CRI-O to lay down the temporary version file (default: /var/run/crio/version)
+
+**--version-file-persist**="": Location for CRI-O to lay down the persistent version file (default: /var/run/crio/version)
```

Comment:
It would be good to add a little verbiage indicating why these files exist, how they are used, and if necessary, the contents.